lhotse/cut/mixed.py

```diff
@@ -704,8 +710,12 @@ def load_features(self, mixed: bool = True) -> Optional[np.ndarray]:
         """
         Loads the features of the source cuts and mixes them on-the-fly.
 
-        :param mixed: when True (default), returns a 2D array of features mixed in the feature domain.
-            Otherwise returns a 3D array with the first dimension equal to the number of tracks.
+        :param mixed: when True (default), the features are mixed together (as defined in
+            the mixing function for the extractor). This could result in either a 2D or 3D
+            array. For example, if all underlying tracks are single-channel, the output
+            will be a 2D array of shape (num_frames, num_features). If any of the tracks
+            are multi-channel, the output may be a 3D array of shape (num_frames, num_features,
+            num_channels).
```

Comment:
any specific reason to go with channels last format?

Reply:
Mainly because I could use ellipsis in the tensor indexing and then the same code would work for both 2D and 3D features.